proto/protovalidate-testing/tests/migrate/field_ignore_empty.migrated.proto

```diff
@@ -21,19 +21,18 @@ import "validate/validate.proto";
 import "buf/validate/validate.proto";
 
 message FieldIgnoreEmpty {
-  int32 x = 1 [(validate.rules).int32.ignore_empty = true,(buf.validate.field).ignore_empty = true];
+  int32 x = 1 [(validate.rules).int32.ignore_empty = true,(buf.validate.field).ignore = IGNORE_IF_UNPOPULATED];
   string y = 2 [
     (validate.rules).int32 = { ignore_empty: false },
     (buf.validate.field).int32 = { }
-    , (buf.validate.field).ignore_empty = false
   ];
   repeated int32 z = 3 [
     (validate.rules).repeated.items = {
       int32: { ignore_empty: true, gt: 10 };
     },
     (buf.validate.field).repeated.items = {
       int32: { gt: 10 }
-      , ignore_empty: true;
+      , ignore: IGNORE_IF_UNPOPULATED;
```

Comment:
Do we not run migrated files through buf format when done?

Also above on line 27 - shouldn't we remove empty validation blocks?

Reply:
We recommend users to run the formatter. We don't however to minimize the diff to just material changes from the migrator.

Empty blocks aren't removed because semantically they replace the entire value when used:

```proto
message Message {
  string value = 1 [
    (buf.validate.field).string.min_len = 3;
    (buf.validate.field).string = { max_len: 5 }; // the previous rule is deleted by this
  ];
```